services/cron/tasks.go

```diff
@@ -90,18 +90,18 @@ func (t *Task) RunWithUser(doer *models.User, config Config) {
 		if err := t.fun(ctx, doer, config); err != nil {
 			if db.IsErrCancelled(err) {
 				message := err.(db.ErrCancelled).Message
-				if err := admin_model.CreateNotice(admin_model.NoticeTask, config.FormatMessage(t.Name, "aborted", doer, message)); err != nil {
+				if err := admin_model.CreateNotice(ctx, admin_model.NoticeTask, config.FormatMessage(t.Name, "aborted", doer, message)); err != nil {
```

Comment:
I'm afraid we cannot get the engine correctly here?

Reply:
@KN4CK3R

Reply:
Doesn't it just use the default context if there is no engine in that context?